Sources/NIO/DatagramVectorReadManager.swift

```diff
@@ -148,9 +178,18 @@ struct DatagramVectorReadManager {
             // Next we extract the remote peer address.
             precondition(self.messageVector[i].msg_hdr.msg_namelen != 0, "Unexpected zero length peer name")
             let address: SocketAddress = self.sockaddrVector[i].convert()
+            
+            // Extract congestion information if requested.
+            let metadata: AddressedEnvelope<ByteBuffer>.Metadata?
+            if reportExplicitCongestionNotifications {
+                let controlMessagesReceived = UnsafeControlMessageCollection(messageHeader: messageVector[i].msg_hdr)
```

Comment:
Why aren't we just using `self.messageVector` like we do on line 179, and removing the extra argument?